obj.go

```diff
@@ -420,6 +448,33 @@ func (O ObjectCollection) Trim(n int) error {
 	return O.drv.checkExec(func() C.int { return C.dpiObject_trim(O.dpiObject, C.uint32_t(n)) })
 }
 
+func (O ObjectCollection) CollAsArrayMap() ([]map[string]interface{}, error) {
+	var array []map[string]interface{}
+	for curr, err := O.First(); err == nil; curr, err = O.Next(curr) {
+		if value, err := O.Get(curr); err != nil {
+			return nil, err
+		} else if value == nil {
+			continue
+		} else {
+			r, _ := value.(*Object).ObjAsMap()
+			array = append(array, r)
+		}
+	}
+	return array, nil
+}
+
+func (coll ObjectCollection) GetFromArrayMap(maps []map[string]interface{}) error {
```

Comment:
Why would sy create such a horrible data structure ([]map[string]interface{}) ?
Don't you know your object's structure beforehand?

Reply:
In my case, often the BE just take the value from FE ad send it to DL.
So just converting json to map and viceversa, could be a best case scenario so the business logic will not be hold by the BE.

Btw even in other situation, i think it could be fine to send data from the json, and so work with the map/map array.

Reply:
Ok, please rebase your code on top of the current master.
That invokes that CollAsArrayMap and ObjAsMap are not needed (we have AsMapSlice and AsMap).
For the remaining  GetFromArrayMap and GetFromMap:
1. rename them to FromMapSlice and FromMap
2. Put nice comment about what they do: Populate the Object/Collection from the map/[]map, and they're short, it's there mostly as an example.